syntax = "proto3";
 
package pb;

service OCRService{
    rpc CreateTextFromFile(FileRequest) returns (FileResponse) {}
}

message FileRequest {
    string file_name = 1;
    bytes file = 2;
}

message FileResponse {
    string message = 1;
    string text = 2;
}

